package.json

```diff
@@ -3,19 +3,27 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@babel/preset-typescript": "^7.18.6",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "mini-css-extract-plugin": "^2.6.1",
+    "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router-dom": "^6.4.2",
     "react-scripts": "5.0.1",
+    "semantic-ui-react": "^2.1.3",
+    "source-map-loader": "^4.0.1",
+    "typescipt": "^1.0.0",
```

Comment:
Do you really need typescript and other dev packets in "dependencies", not in "devDependencies"? If you deploy your application on prod, this approach installs unnecessary data on production server.